themes/default/content/blog/examining-recommended-practices-code-placement-and-stacks/index.md

```diff
@@ -0,0 +1,148 @@
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - scott-lowe
+    - christian-nunciato
+    - aaron-kao
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - development-environment
+    - cloud-engineering
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+This is the first in a series of blog posts that explores how a fictional company---Zephyr Archaeotech Emporium---uses Pulumi to manage their online retail store. This post explores a couple common questions that users ask when working with Pulumi; specifically, where should I store my Pulumi code? And how do I support multiple environments with Pulumi? This post will provide some guidance and recommended practices around these topics, using Zephyr and their online store as the use case.<!--more-->
+
+The ultimate goal of this series is to discuss recommended practices for using Pulumi to manage a fairly complex containerized application. However, it's important to note that these recommended practices will emerge over the course of the series---not all immediately, and not all in the beginning. This is a deliberate decision to allow you to see how Zephyr's use of Pulumi evolves as the company grows and their online retail store application changes to accommodate their growth.
+
+Here are links to all the blog posts in the series (entries below that are not linked are planned but haven't yet been published---this will get updated as new posts are published): <!--The names of these posts and the order in which we publish them is mutable-->
+
+**Examining Recommended Practices: Code Placement and Stacks** (this post)
+
+Examining Recommended Practices: Developer Stacks and Git Branches
+
+Examining Recommended Practices: Local Testing with Pulumi
+
+Examining Recommended Practices: Structuring Pulumi Projects
+
+Examining Recommended Practices: Moving Infrastructure State Between Projects
+
+Examining Recommended Practices: Tying Stacks Together with Stack References
+
+Examining Recommended Practices: Evolving the Application
+
+Examining Recommended Practices: Adding Pulumi Deployments
+
+Examining Recommended Practices: Refactoring for Reuse
+
+## Setting up the scenario
+
+Zephyr is short for Zephyr Archaeotech Emporium, the fictional company in our scenario. Zephyr is an online retailer that specializes in the sale of "rare arcane artifacts and their replicas." Over the past few years, the company has experienced a significant increase in its online presence, making it easier for customers to purchase unique and mysterious items. Zephyr's collection includes a variety of rare and unusual objects that are difficult to find elsewhere, making it a popular destination for collectors, enthusiasts, and adventurers.
```

Comment:
Nit, I keep stumbling over the word "their" in these variations -- suggest removing it:

```suggestion
Zephyr is short for Zephyr Archaeotech Emporium, the fictional company in our scenario. Zephyr is an online retailer that specializes in the sale of "rare arcane artifacts and replicas." Over the past few years, the company has experienced a significant increase in its online presence, making it easier for customers to purchase unique and mysterious items. Zephyr's collection includes a variety of rare and unusual objects that are difficult to find elsewhere, making it a popular destination for collectors, enthusiasts, and adventurers.
```

Reply:
Fixed in 3b48f0ca